# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/libwebrtc/call/audio_receive_stream.h
# Commit: 26913ec8db0b
# Full Hash: 26913ec8db0b350b29145e9502136b29efdeccbc
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Reimplement ssrc/csrc stats cache. r=ng
#   
#   The cache can be a lot simpler than the old one, since libwebrtc handles
#   pruning old entries for us, and since we're now telling libwebrtc when we
#   render frames (so we don't have to make our own stats with guesses of when
# ==============================================================================

diff -r 18d3bc91b71e -r 26913ec8db0b third_party/libwebrtc/call/audio_receive_stream.h
--- a/third_party/libwebrtc/call/audio_receive_stream.h	Thu Feb 25 13:07:39 2021 -0600
+++ b/third_party/libwebrtc/call/audio_receive_stream.h	Mon Mar 15 12:19:57 2021 -0500
@@ -200,9 +200,6 @@
 
   virtual std::vector<RtpSource> GetSources() const = 0;
 
-  virtual void InsertAudioLevelForContributingSource(
-      const RtpPacketInfos& packet_infos) = 0;
-
  protected:
   virtual ~AudioReceiveStream() {}
 };
